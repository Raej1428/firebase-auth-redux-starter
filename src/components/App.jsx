/* @flow */
import React from 'react';

const App = () => (
  <div className="sans-serif">
    <p>
      Hello from app component!
    </p>
  </div>
);

export default App;
