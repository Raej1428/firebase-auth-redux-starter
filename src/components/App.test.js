/* eslint-disable */

import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow'
import App from './App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});

it('matches snapshot', () => {
  const shallow = new ShallowRenderer();
  shallow.render(<App />);
  const result = shallow.getRenderOutput();
  expect(result).toMatchSnapshot();
})
