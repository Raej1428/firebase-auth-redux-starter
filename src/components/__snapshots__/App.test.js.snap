// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="sans-serif"
>
  <p>
    Hello from app component!
  </p>
</div>
`;
